### PR TITLE
Fix #942 - Truncate Alias Label

### LINKED ISSUE #942: Longer alias labels are cut off
#### Build:
- Stage;

#### Affected Platforms:
- Windows 10;

#### Browsers tested on:
- Firefox, latest version;

#### Prerequisites:
- Be signed in into a Firefox Relay account with an active subscription;
- Have a few aliases created;
- Have the stage version of the Relay add-on installed;

#### Steps to reproduce:
1. Go to https://stage.fxprivaterelay.nonprod.cloudops.mozgcp.net/accounts/profile/ ;
2. Click on the “Add account name” bar above any created alias;
3. Write a long name;

#### Expected result:
- The label is displayed in its entirety;

#### Actual result:
- The label is cut off;

#### Notes:
- N/A;

labelcut1 1
labelcut1 2
labelcut1 3

### PR DESCRIPTION
**Pull request by codemist**
**Preview:**
image
image

### CODE REVIEW

File: static/scss/partials/main.scss [hunk 1]
```diff
@@ -1080,7 +1080,7 @@ input:focus::placeholder {
 
 .relay-email-address-label:focus,
 input.input-has-error {
-  width: 200px !important;
+ // width: 200px !important;
```
Comment (groovecoder) [inline static/scss/partials/main.scss:1083]: nit: let's go ahead and remove the line rather than comment.

Comment (codemist) [reply static/scss/partials/main.scss]: Done!

File: static/scss/partials/main.scss [hunk 2]
```diff
@@ -1032,6 +1032,7 @@ input::placeholder,
 .relay-email-address-label:placeholder-shown {
   color: var(--grey50);
   opacity: 1;
+  width: 192px !important;
```
Comment (maxxcrawford) [inline static/scss/partials/main.scss:1035]: Is the `!important` tag necessary here? 
```suggestion
  width: 192px !important;
```

Comment (maxxcrawford) [reply static/scss/partials/main.scss:1035]: Also, there is a token, `$layout-2xl` that is `192px` wide. We might use that instead? (That may not be the most semantically correct use of the token)

**Review by maxxcrawford — CHANGES_REQUESTED** [review]

File: static/scss/partials/main.scss (hunk 2 above)
Comment (codemist) [reply static/scss/partials/main.scss:1035]: Hmm, the `192px` is more of a Magic Number fix for the length of the placeholder, and from my understanding `$layout-2xl` should be reserved for spacing between elements.

Comment (maxxcrawford) [reply static/scss/partials/main.scss]: Well reasoned. Good to go!

**Review by maxxcrawford — CHANGES_REQUESTED** [review]
Good to go as soon as you resolve the conflicts! 🙈

**Review by maxxcrawford — APPROVED** [review]

**Pull request merged**